app/controllers/concerns/hyrax/works_controller_behavior.rb

```diff
@@ -8,6 +8,16 @@ module WorksControllerBehavior
       layout :decide_layout
     end
 
+    class_methods do
+      # We don't want the breadcrumb action to occur until after the concern has
+      # been loaded and authorized
+      def curation_concern_type=(curation_concern_type)
+        super
+        before_action :build_breadcrumbs, only: [:edit, :show, :new]
```

Comment:
I don't know why this line must be in this method.  It seems to me that the place for this line should remain in BreadcrumbsForWork